upup/pkg/fi/cloudup/awstasks/launchconfiguration.go

```diff
@@ -286,7 +288,12 @@ func (_ *LaunchConfiguration) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *La
 		if awsup.AWSErrorCode(err) == "ValidationError" {
 			message := awsup.AWSErrorMessage(err)
 			if strings.Contains(message, "not authorized") || strings.Contains(message, "Invalid IamInstance") {
-				return fmt.Errorf("IAM instance profile not yet created/propagated (original error: %v)", message)
+				if attempt > maxAttempts {
+					return fmt.Errorf("IAM instance profile not yet created/propagated (original error: %v)", message)
+				}
+				glog.Infof("waiting for IAM instance profile %q to be ready", fi.StringValue(e.IAMInstanceProfile.Name))
+				time.Sleep(10 * time.Second)
```

Comment:
No, IAM propagation is slow, and the last thing we want to do is start hitting rate limits.

Also, this is the same as the task backoff we had before.